woff1/tools/validator/validator.py

```diff
@@ -17,9 +18,13 @@
 import zlib
 import optparse
 import codecs
-from cStringIO import StringIO
+from io import BytesIO
 from xml.etree import ElementTree
 from xml.parsers.expat import ExpatError
+try:
+    basestring
+except NameError:
+    basestring = str
```

Comment:
Provide Python 3 fallback from `basestring` type, which only exists in Python 2.